plugins/action_update_review_status/src/updateReviewStatuses.ts

```diff
@@ -60,6 +60,8 @@ const updateReviewStatuses: ActionPluginType = async ({
     const reviews = (await db.getAssociatedReviews(applicationId, stageNumber))
       // Ignore all "Discontinued" reviews
       .filter(({ reviewStatus }) => reviewStatus !== ReviewStatus.Discontinued)
+      // Ignore current review
+      .filter(({ reviewId: id }) => id !== reviewId)
```

Comment:
Great, but tiny bit more efficient to do it in one filter pass:
```
.filter(({ reviewStatus, reviewId: id }) => reviewStatus !== ReviewStatus.Discontinued &&
     id! == reviewId)
```

Reply:
I just did like this so it's separated and clearer... I think will keep like that still